### DISCUSSION TITLE
Setup stopped at "EFI stub: Loaded initrd from command line option"

### DISCUSSION THREAD

**Discussion by calvss** [antonym/netboot.xyz#1201] [Q&A]
Hi, I'm trying to install Debian 11 on an Oracle Cloud VM.Standard.E2.1.Micro, but the install gets stuck on "EFI stub: Loaded initrd from command line option".

- I started with booting an Ubuntu 20.04 minimal image to download netboot.xyz.efi and put it in /boot/efi/.
- I open a serial console and reboot to get access to the boot menu and select "Boot maintenance manager -> Boot from file -> netboot.xyz.efi"
- The selection menu appears and I select "Linux network installs -> Debian 11"
- The install sequence downloads 2 vmlinuz and initrd, prints their hashes, then prints "EFI stub: Loaded initrd from command line option"

The install process gets stuck here and I'm not sure why. I don't know where to start troubleshooting this issue, would anyone know?

More info:
I tried the install with a different distro,  selecting Ubuntu 20.04 instead of Debian 11 and it seems like the install process gets stuck after downloading the ubuntu-squash files and printing their hashes.

**Comment by etnperlong** [comment]
The same issue, need help plz.
